src/humanpy/hum_kin1.py

```diff
@@ -0,0 +1,263 @@
+PACKAGE = 'humanpy'
+import numpy
+import rospy
+import rospkg
+import collections
+from scipy import signal
+from tf import transformations, LookupException, ConnectivityException, ExtrapolationException
+import humanpy
+
+BASE_FRAME = '/map'
+KIN_FRAME = '/head/skel_depth_frame'
+
+    
+class Orhuman(object):
+    def __init__(self, id, env, **kw_args):
+        assert id != ''
+        self.id = id
+        self.enabled = True
+        _,self.body = humanpy.initialize(sim=False, user_id=id, env=env)
+        self.initialDOFvalues = self.body.GetDOFValues()
+        self.currentDOFvalues = self.body.GetDOFValues()        
+        self.starting_angle = dict()
+        self.last_updated = rospy.get_rostime().secs
+        self.collshl = collections.deque(maxlen=300)
+        self.collsh2l = collections.deque(maxlen=300)
+        self.collsh3l = collections.deque(maxlen=300)
+        self.collell = collections.deque(maxlen=300)
+        self.collshr = collections.deque(maxlen=300)
+        self.collsh2r = collections.deque(maxlen=300)
+        self.collsh3r = collections.deque(maxlen=300)
+        self.collelr = collections.deque(maxlen=300)
+        self.collthl = collections.deque(maxlen=300)
+        self.collcal = collections.deque(maxlen=300)
+        self.collthr = collections.deque(maxlen=300)
+        self.collcar = collections.deque(maxlen=300)
```

Comment:
Where did the magic number `300` come from? Also, it would be nice to make this an argument (or at least a local variable) to avoid repeating the same constant this many times.